### DISCUSSION TITLE
Support for snapdragon/arm64

### DISCUSSION THREAD

**Discussion by Olemann96** [OrcaSlicer/OrcaSlicer#7821] [General]
Will it be support for snapdragon pc?      BambuStudio works great but OrcaSlicer give error about OpenGL 2.0

**Comment by dragon2611** [comment]
There's an OpenGL compatibility pack in the Microsoft store (It should be by Microsoft themselves) if you install that then Orca seems to work ok. (or at least it did in my snapdragon X plus based laptop)

**Reply by RyanEwen** [reply]
This appears to have been preinstalled for me and Orca does work on my machine, though I would really like to see an ARM native version with better 3d acceleration

**Comment by Basti77** [comment]
Does the Camera Stream work for you?

It doesn't work for me on the device with the Snapdraon CPU.

**Reply by RyanEwen** [reply]
No, it seems that the camera doesn't work. I hadn't noticed until you asked since I use SimplyPrint for the most part

**Comment by Basti77** [comment]
OK then it's one more thing that doesn't work. 
I always have to use the Bambu studio mobile app to see what my printer is doing.
Please create a real ARM64 version.

**Comment by RyanEwen** [comment]
PSA: Devs who don't have access to a Snapdragon machine but do have access to a Mac with an M-series processor can use Parallels to develop. Parallels uses the ARM build of Windows on M-series Macs.

**Comment by Basti77** [comment]
I think you have to be friendlier than that: 
I'm afraid they ignore such requests if you get snotty. 

I myself tried to build the project for ARM64 with the help of Chat gpt. 
Frankly, I'm not a developer and know at best half of what I'm doing. the project was compiled but the exe didn't run. 
I'm not surprised because I didn't really know what I was doing with all the libs. 

if it's only about testing the builds, there will surely be test users. for my own benefit I would also contact them.

**Comment by RyanEwen** [comment]
Heads-up, another option has been found for devs without access to Windows on ARM: https://learn.microsoft.com/en-us/windows/arm/create-arm-vm
